src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs

```diff
@@ -85,7 +85,7 @@ at com.android.dx.command.Main.main(Main.java:106)
 
 		public virtual string DefaultErrorCode => "JAVA0000";
 
-		public string WorkingDirectory { get; set; }
+		public new string WorkingDirectory { get; set; }
```

Comment:
Can we remove the `WorkingDirectory` property instead and just use the one on the base class?